### DISCUSSION TITLE
Periodic backups in v6.0

### DISCUSSION THREAD

**Discussion by n-bilalis** [ravendb/ravendb#17628] [General]
RavenDB is an excellent product and I can't really complain, since I use it with a Community license. That being said...

I've got the notification for the new version and, although I was skeptical at first, I decided to go ahead and do it. It didn't take long to realize that Periodic Backups are no longer available with the Community license in v6.0. I have to respect this, as a business decision, but I wish it was communicated to me, somehow, before doing the upgrade, so that I would be spared the hustle to downgrade back to v5.4.

It wasn't easy for me to convince my team to use RavenDB, since no one in my team knew about it, and now I get even more pressure to change to something like PostgreSQL. Their concern is whether v5.4 will still get updates.

(Purchasing a "proper" license is no option for us. We work for a public sector educational institute, and we have to use free software in our projects.)

**Comment by DejanMilicic** [comment]
Version 5.4 is LTS. After the next LTS version is out, 5.4 will be supported for another year.
This means that you can count on at least two more years of support for 5.4

**Reply by n-bilalis** [reply]
That's good to know, thank you
